src/app.ts

```diff
@@ -143,6 +144,12 @@ async function app() {
       process.exit(0);
     }
     config = readConfig(originalInput);
+    if (config.plugins) {
+      await PluginManager.processPluginConfiguration(
+        config.plugins,
+        path.join(input, '..', 'plugin')
```

Comment:
The PR description says to put plugins in `input/plugins`, but this is looking at `input/plugin`.  I expect we want the plural.

I also wonder if `input/plugins` is the right path anyway.  Plugins are more software than input; as I think of `input` as containing _data_.  In addition, I think the name needs to be SUSHI specific (to distinguish from plugins for, say, the IG Publisher).

So I'd suggest we consider something like `${projectRoot}/sushi-plugins/` instead.